source/Calamari.Common/Plumbing/Extensions/AesEncryption.cs

```diff
@@ -17,9 +32,10 @@ public class AesEncryption
 
         readonly byte[] key;
 
-        public AesEncryption(string password)
+        public AesEncryption(string password, int keySizeBits)
```

Comment:
Making the key size configurable allows us to upgrade the 3 components separately.

Reply:
The flexibility of this is nice, I wonder if this might open up the opportunity for the wrong one to be used in the wrong place.

It might be better to use inheritance here

keep `AesEncryption` abstract, and impliment the `Bootstrapper` and `Script` one passing through these key sizes required.

Reply:
Changed to factory methods for the scenarios and making the encryption key public on the encryptor